contrib/all/Dockerfile.apiserver

```diff
@@ -1,6 +1,6 @@
 ARG IMAGE
 
-FROM golang:1.13 as builder
+FROM keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/library/golang:1.16-alpine as builder
```

Comment:
Two things: First, you want `1.16-alpine3.13`. The newest Alpine has problems with the Docker version that our Concourse uses: https://gitlab.alpinelinux.org/alpine/aports/-/issues/12519

Second, I'd recommend not hardcoding our internal Docker Hub mirror here. Leave the references to the original Docker Hub images here, and use `ci/shared/task-inject-dockerhub-mirror.yaml` in the pipeline (you can look at the `build` task in the Keppel pipeline for a usage example).

Reply:
Thanks! I modified the image reference. The task injection seems like a hassle. I keep it in mind :)